packages/lit-dev-content/src/components/litdev-search.ts

```diff
@@ -289,6 +290,18 @@ class LitDevSearch extends LitElement {
     document.location = searchUrl.href as unknown as Location;
     this.input.value = '';
     this.searchText = '';
+
+    // On mobile we manually close the nav drawer, otherwise the drawer remains
+    // open when navigating between fragment identifiers.
+    const navDrawer = document.querySelector(
+      'mwc-drawer#mobileDrawer'
+    ) as Drawer;
+    if (navDrawer) {
+      navDrawer.open = false;
+    }
+
+    // Rerender to visually clear the value on the input.
+    this.requestUpdate();
```

Comment:
Instead of calling `requestUpdate` here, could you make `searchText` a `@state` property so that when you set it above, it automatically re-renders?

Reply:
Great suggestion. I don't know why I didn't reach for the declarative approach first. Thank you!